packages/components/Organisms/HeroBanner/HeroBanner.vue

```diff
@@ -180,6 +180,7 @@ export default {
     'xxxl': ('pos': 'right', 'v-offset': $rpl-hero-banner-background-graphic-v-offset-xxl),
     '2660px': ('pos': 'right')
   ) !default;
+  $rpl-hero-banner-title-word-wrap: break-word !default;
```

Comment:
I also not thinking of a setting for this, any reason need this to be configurable?